djangocms_versioning/admin.py

```diff
@@ -363,18 +363,10 @@ def unpublish_view(self, request, object_id):
         )) + '?grouper=' + str(version.grouper.pk)
         return redirect(url)
 
-    def edit_redirect_view(self, request, object_id):
-        """Redirects to the admin change view and creates a draft version
-        if no draft exists yet.
-        """
-        # This view always changes data so only POST requests should work
-        if request.method != 'POST':
-            return HttpResponseNotAllowed(['POST'], _('This view only supports POST method.'))
-
+    def _get_edit_redirect_version(self, request, object_id):
         version = self.get_object(request, unquote(object_id))
         if version is None:
-            return self._get_obj_does_not_exist_redirect(
-                request, self.model._meta, object_id)
+            return
```

Comment:
This has lost the "_get_obj_does_not_exist_redirect" message. Obviously now this is a helper method for the view it can't just return the redirect here. Is it ok that it will always return a 404 rather than an admin error and message in this instance?